commercetools-models/src/main/java/io/sphere/sdk/products/search/ProductCategoriesIdTermFacetSearchModel.java

```diff
@@ -24,6 +24,9 @@
     @Override
     ProductCategoriesIdTermFacetSearchModel<T> withAlias(final String alias);
 
+    @Override
+    ProductCategoriesIdTermFacetSearchModel<T> countingProducts(final Boolean isCountingProducts);
```

Comment:
I was doubting how we did it in the past, I want to be consistent. But I couldn't remember any situation. In any case, I think it reads better *without* the `is`.